DataManagementSystem/Client/DataManager.py

```diff
@@ -616,7 +616,7 @@ def __isSEInList( self, seName, seList ):
         seSet.add( res['Value'] )
     return self.__getSERealName( seName ).get( 'Value' ) in seSet
 
-  def __replicate( self, lfn, destSE, sourceSE = '', destPath = '', localCache = '' ):
+  def __replicate( self, lfn, destSEName, sourceSEName = '', destPath = '', localCache = '' ):
```

Comment:
Okay so digging more, I see this comes from a bad merge of this PR https://github.com/DIRACGrid/DIRAC/commit/ee43960372dcf54dd1f765ea79a3e5bd607d3639 
I hope the rest was properly merged.